inscriptions/ISic000488.xml

```diff
@@ -192,14 +197,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>tate et iustitia pr<choice><reg>ae</reg><orig>e</orig></choice>di<supplied reason="lost">tus</supplied>
+                    <lb n="2"/><supplied reason="lost">e</supplied>t <g ref="#interpunct">·</g> singulari <g ref="#interpunct">·</g> sincerita<supplied reason="lost">te</supplied>
```

Comment:
I think one should put `<gap reason="lost" extent="unknown" unit="character"/>` at the beginning and end of every line of this fragment, since there is no reason to think it intact at either end.